packages_rs/nextclade-web/src/io/fetchDatasetsIndex.ts

```diff
@@ -6,7 +6,11 @@ import urljoin from 'url-join'
 import { Dataset, DatasetsIndexV2Json } from 'src/algorithms/types'
 import { axiosFetch } from 'src/io/axiosFetch'
 
-const DATA_FULL_DOMAIN = process.env.DATA_FULL_DOMAIN ?? '/'
+var DATA_FULL_DOMAIN = process.env.DATA_FULL_DOMAIN ?? '/'
```

Comment:
Let's change this `var` to `let`.

```suggestion
let DATA_FULL_DOMAIN = process.env.DATA_FULL_DOMAIN ?? '/'
```